src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package com.softserve.actent.service.impl;
+
+import com.softserve.actent.model.entity.Review;
+import com.softserve.actent.repository.ReviewRepository;
+import com.softserve.actent.service.ReviewService;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ReviewServiceImpl implements ReviewService {
+
+    @Autowired
+    ModelMapper modelMapper;
+
+    @Autowired
+    ReviewRepository reviewRepository;
```

Comment:
Great one, but check in which cases its better to use autowiring through constructor whether then like a field (intellij might hint you that ;)

Reply:
I will find out, thank you.